src/Nox.Lib/Infrastructure/Persistence/EntityDbContextBase.cs

```diff
@@ -54,6 +54,11 @@ public override async Task<int> SaveChangesAsync(CancellationToken cancellationT
 
         public virtual async Task<long> GetSequenceNextValueAsync(string sequenceName)
         {
+
+            var t = new List<int>();
```

Comment:
I guess this is a leftover from some testing?

Reply:
yes will remove it